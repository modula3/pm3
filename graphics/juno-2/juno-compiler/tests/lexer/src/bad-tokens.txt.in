foo?bar
_foo
2.0e
2.0e foo
(* unclosed comment
"unclosed text
"Unknown escape char: \a"
"Bad escape char: \777"
"Bad escape char: \00x"
"Bad escape char: \0"
"Bad escape char: \
