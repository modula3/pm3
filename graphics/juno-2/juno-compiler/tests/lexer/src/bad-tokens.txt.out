Each line you enter will be parsed into tokens.

> foo?bar
Id(foo)
Bad Token! (Illegal initial character)
-> Read = <>
-> Left = <?bar>

> _foo
Bad Token! (Illegal initial character)
-> Read = <>
-> Left = <_foo>

> 2.0e
Bad Token! (Illegal number)
-> Read = <2.0e>
-> Left = <>

> 2.0e foo
Bad Token! (Illegal number)
-> Read = <2.0e>
-> Left = < foo>

> (* unclosed comment
Bad Token! (Unclosed comment)
-> Read = <(* unclosed comment>
-> Left = <>

> "unclosed text
Bad Token! (Unclosed text literal)
-> Read = <"unclosed text>
-> Left = <>

> "Unknown escape char: \a"
Bad Token! (Illegal \ escape)
-> Read = <"Unknown escape char: \>
-> Left = <a">

> "Bad escape char: \777"
Bad Token! (Illegal \ escape)
-> Read = <"Bad escape char: \777>
-> Left = <">

> "Bad escape char: \00x"
Bad Token! (Illegal \ escape)
-> Read = <"Bad escape char: \00>
-> Left = <x">

> "Bad escape char: \0"
Bad Token! (Illegal \ escape)
-> Read = <"Bad escape char: \0>
-> Left = <">

> "Bad escape char: \
Bad Token! (Illegal \ escape)
-> Read = <"Bad escape char: \>
-> Left = <>

> ^D
